src/site/layouts/home.drupal.liquid

```diff
@@ -0,0 +1,139 @@
+{% include "src/site/includes/header.html" %}
```

Comment:
I think having this include is right—at least it follows what I'm seeing in like `src/site/layouts/page.html` and friends—but should it have the `with drupalTags = true` bit like the other `.drupal.liquid` files that include this have?

I'm not actually sure what that `drupalTags` does, to be honest, but figured I should ask.